gamestonk_terminal/economy/cnn_view.py

```diff
@@ -31,23 +34,24 @@ def fear_and_greed_index(indicator: str, export: str):
         Safe Heaven Demand, and Index.
     export : str
         Export plot to png,jpg,pdf file
+    external_axes : Optional[List[plt.Axes]], optional
+        External axes (1 axis is expected in the list), by default None
     """
-    fig = plt.figure(figsize=plot_autoscale(), dpi=PLOT_DPI)
+    fig, ax = plt.subplots(figsize=plot_autoscale(), dpi=PLOT_DPI)
```

Comment:
since we do pass external exes here, let's create a new fig, ax only `if external_axes is None`

although i don't like this command and would suggest to ditch it